cross-chain-atomic-swap-cordapp/src/test/kotlin/jp/co/layerx/cordage/crosschainatomicswap/TestUtils.kt

```diff
@@ -5,8 +5,21 @@ import net.corda.testing.core.BOB_NAME
 import net.corda.testing.core.CHARLIE_NAME
 import net.corda.testing.core.DUMMY_BANK_A_NAME
 import net.corda.testing.core.TestIdentity
+import java.util.*
 
 val ALICE = TestIdentity(ALICE_NAME)
 val BOB = TestIdentity(BOB_NAME)
 val CHARLIE = TestIdentity(CHARLIE_NAME)
 val DUMMY_BANK_A = TestIdentity(DUMMY_BANK_A_NAME)
+
+fun readConfig(key: String): String {
+    val properties = Properties()
+    val root = java.lang.System.getProperty("user.dir")
+
+    val path = java.nio.file.Paths.get("$root/src/test/resources/config.conf")
```

Comment:
You can resolve path with `javaClass.classLoader.getResource("config.conf")`

Reply:
I updated the code, but I still need to have one part of the logic for when the code is running in test mode (so when you don't have a /cordapps/config/config.conf file) and one when you are running it 'normally' .

Reply:
I understand what you are doing. This is just a kind of code review.
Even if you run tests with `javaClass.classLoader.getResource("config.conf")`, you can access the config.conf in the src/test/resources directory.

Reply:
So in fact what I have now in Utils.kt is correct no ?

Reply:
Ah nooooo it's in TestUtils.kt not Utils.kt , hence my confusion x)

Reply:
Sorry fixing it now

Reply:
Done, somehow I could not use `javaClass.classLoader.getResource("config.conf")` but had to change it to `object{}.javaClass.classLoader.getResource("config.conf")` . Not sure why .